app/controllers/provider/passwords_controller.rb

```diff
@@ -47,6 +47,10 @@ def update
     end
   end
 
+  def request_password_reset
```

Comment:
what about calling it just `reset` ? it's inside the PasswordsController so no need to say that it is the password what is being reset... and the `request` word... all methods here are requests 🤔 it is like naming `update` as `request_password_update`, simply not needed 😄